iroha_client/tests/asset_amount_should_be_the_same_on_a_recently_added_peer.rs

```diff
@@ -16,7 +16,8 @@ mod tests {
         let (rt, network, mut iroha_client) = <Network>::start_test_with_runtime(4, 1);
         let pipeline_time = Configuration::pipeline_time();
 
-        thread::sleep(pipeline_time * 3);
+        thread::sleep(pipeline_time * 2);
+        iroha_logger::info!("Started");
```

Comment:
Are these and below - debug logs? If so then please remove, if not then ok.

Reply:
Should we remove those for tests?

Reply:
I would remove, but that's not critical - so up to you